docs/declarative-customization/column-formatting.md

```diff
@@ -864,11 +864,11 @@ You can use the following predefined classes for several common scenarios.
 > [!NOTE]
 > The icons shown above for the `sp-field-severity` classes are **NOT** part of the class. Only the background color is included. Icons can be added by using the `iconName` attribute.
 
-In addition to the classes listed above, the classes (such as the theme color, typography, grid system, etc.) defined by the Office UI Fabric can be used. For details, see the [Fabric website](https://dev.office.com/fabric#/styles/colors). 
+In addition to the classes listed above, the classes (such as the theme color, typography, grid system, etc.) defined by the Office UI Fabric can be used. For details, see the [Fabric website](https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/products). 
 
 ### Predefined icons
 
-You can use predefined icons from Office UI Fabric. For details, see the [Fabric website](https://dev.office.com/fabric#/styles/icons). 
+You can use predefined icons from Office UI Fabric. For details, see the [Fabric website](https://developer.microsoft.com/en-us/fluentui#/styles/web/icons). 
```

Comment:
Same comment as above with locale.

Reply:
Done. Thank you for the review comments.